index.html

```diff
@@ -3,22 +3,225 @@
     <head>
```

Comment:
This is more of a style comment, but I would see if you could make your page look more like the design.  I know that with a page that looks like this, a designer would come back and want you to tighten things up a bit.